update-imported-docs.sh

```diff
@@ -49,12 +43,68 @@ while read line || [[ -n ${line} ]]; do
 done <_data/overrides.yml
 
 # refdoc munging
+pushd .
 cd _includes/v$VERSION
-find . -name '*.html' -type f -exec sed -i -e '/<style>/,/<\/style>/d' {} \;
-find . -name '*.html' -type f -exec sed -i -e "s/http:\/\/kubernetes.io\/v$VERSION//g" {} \;
-cd ..
-cd ..
+  # These are included in other files, so strip the DOCTYPE
+   find . -name '*.html' -type f -exec sed -i -e "s/<!DOCTYPE html>//g" {} \;
+
+  # Format html
+  find . -name '*.html' -type f -exec sed -i -e '/<style>/,/<\/style>/d' {} \;
+  find . -name '*.html' -type f -exec sed -i -e "s/http:\/\/kubernetes.io\/v$VERSION//g" {} \;
+popd
+
+pushd .
+cd docs/api-reference
+  # Format html
+  find . -name '*.html' -type f -exec sed -i -e '/<style>/,/<\/style>/d' {} \;
+  find . -name '*.html' -type f -exec sed -i -e "s/http:\/\/kubernetes.io\/v$VERSION//g" {} \;
+  find . -name '*.html' -type f -exec sed -i -e '1 i\
+---' {} \;
+  find . -name '*.html' -type f -exec sed -i -e '1 i\
+---' {} \;
+
+  # Strip the munge comments
+  find . -name '*.md' -type f -exec sed -i -e '/<!-- BEGIN MUNGE: IS_VERSIONED -->/,/<!-- END MUNGE: IS_VERSIONED -->/d' {} \;
+  find . -name '*.md' -type f -exec sed -i -e '/<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->/,/<!-- END MUNGE: UNVERSIONED_WARNING -->/d' {} \;
+
+  # Add the expected headers to md files
+  find . -name '*.md' -type f -exec sed -i -e '1 i\
+---' {}  \;
+  find . -name '*.md' -type f -exec sed -i -e '1 i\
+---' {}  \;
+
+  # Fix for bug in 1.3 release
+  find . -name '*.md' -type f -exec sed -i -e "s/vv1.3.0-beta.0/v1.3/g" {} \;
+popd
+
+pushd .
+cd docs/user-guide/kubectl
+  # Strip the munge comments
+  find . -name '*.md' -type f -exec sed -i -e '/<!-- BEGIN MUNGE: IS_VERSIONED -->/,/<!-- END MUNGE: IS_VERSIONED -->/d' {} \;
+  find . -name '*.md' -type f -exec sed -i -e '/<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->/,/<!-- END MUNGE: UNVERSIONED_WARNING -->/d' {} \;
+
+# Add the expected headers to md files
+  find . -name '*.md' -type f -exec sed -i -e '1 i\
+---' {} \;
+  find . -name '*.md' -type f -exec sed -i -e '1 i\
```

Comment:
You're running find so much that you could:
- Write a function to make it simpler (less redundant)
- run find/sed once but apply multiple -e (one for each pattern)